encryption/test/index.test.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const { expect } = require('chai');
```

Comment:
comment not relevant to this file

Why did we leave out the `test/types` directory? Those will still be relevant, particularly because we won't intend to make TS changes in this effort IIUC